client/src/main/java/org/asynchttpclient/netty/channel/NonBlockingSemaphore.java

```diff
@@ -0,0 +1,35 @@
+package org.asynchttpclient.netty.channel;
```

Comment:
Please add package header, granting copyright to "The AsyncHttpClient" project, just like in `ChannelManager` but with inception year 2017.